### PR TITLE
Adding an input to allow insecure SSL/TLS URLS

### PR DESCRIPTION
**Pull request by ScottMillard**
This adds an input defaulted to `false` that when set to `true` will add the `--insecure` flag to the curl statement to allow insecure SSL URLs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ steps:
       # Basic auth login password pair.
       # Format: `login:password`
       basic-auth: "login:password" # Optional, default is empty
+      allow-insecure: "false" # Optional, default is false
```
Comment (Jtalk) [inline README.md:26]: This should probably not be quoted, like the rest of the booleans above.

**Review by Jtalk — CHANGES_REQUESTED** [review]
Thanks for your contribution! 

If I may ask you to:
- run build and commit in the updated dist.js so that CI could run properly;
- add a test to cover the expected failure when accessing badssl with the new param left as default, to make sure GitHub won’t misinterpret “false” in action.yaml as “non-empty string, must be True” (been there, done that, not a picnic). 

Thanks!

**Comment by tobiasdenzler-nag** [comment]
Is there a reason why this one is closed? I would be happy to see this functionality.

**Comment by Jtalk** [comment]
Hey @tobiasdenzler-nag, Scott gave no reason for closing it, but I assume his circumstances had simply changed, and he was no longer in a position to finalise it.

I agree this functionality could be useful. Would you be interested in taking over this PR?

**Pull request closed without merging**